### PR TITLE
Fixes issue #4612

### LINKED ISSUE #4612: "Connect" link for Java console should be disabled when pod is still pending
The Connect link in the pod template shows up when the pod is still pending. Clicking it takes you to the Java console with a blank page and 503 service unavailable errors in web developer console.

We should probably show a reasonable error message as well in the Java console if we get a 503.

screen shot 2015-09-10 at 8 12 33 am

@gashcrumb

### PR DESCRIPTION
**Pull request by gashcrumb**
Fixes #4612

### CODE REVIEW

File: assets/app/scripts/extensions/javaLink.js [hunk 1]
```diff
@@ -62,6 +62,7 @@ angular.module('openshiftConsoleExtensions', ['openshiftConsole'])
                                  });
         window.location.href = targetURI.toString();
       };
+      console.log("added");
```
Comment (liggitt) [inline assets/app/scripts/extensions/javaLink.js]: remove debug logging

**Comment by gashcrumb** [comment]
@liggitt PTAL when you get a chance, thanks!

File: pkg/assets/bindata.go [hunk 2]
```diff
@@ -17239,7 +17239,7 @@ title:g,
 returnTo:e
 });
 window.location.href = i.toString();
-};
+}, console.log("added");
```
Comment (liggitt) [inline pkg/assets/bindata.go]: needs rebuild to pick up removal

**Comment by gashcrumb** [comment]
@liggitt done, sorry 'bout that...

**Comment by liggitt** [comment]
lgtm [merge]

**Comment by liggitt** [comment]
@gashcrumb needs java bindata rebuilt:
https://travis-ci.org/openshift/origin/jobs/79871210#L1278

**Comment by gashcrumb** [comment]
@liggitt Strange, running `install_assets.sh` and `build_assets.sh` doesn't cause any changes, should I rebase it maybe?

**Comment by liggitt** [comment]
not sure, clean/install/build?

**Comment by gashcrumb** [comment]
@liggitt there we go, `git clean` to the rescue...

**Comment by liggitt** [comment]
go ahead and squash while you're at it

**Comment by liggitt** [comment]
lgtm, [merge]

**Pull request merged**